README.md

```diff
@@ -148,19 +148,70 @@ The parser has five parameters but only two of them are compulsory, the other th
 This is how it is run only specifying the required parameters:
 
 ```sh
-(venv)$ ppython3 modules/CRISPR.py -s 1.7.5 -c crispr_cell_lines.tsv -o crispr-18-11-2020.json
+(venv)$ python3 modules/CRISPR.py -s 1.7.5 -c crispr_cell_lines.tsv -o crispr-18-11-2020.json
 ```
 
-### PhenoDigm - What version should I run?
-The PhenoDigm parser used to generate the 19.04 release data is the [solr_phenodigm_1904](https://github.com/opentargets/evidence_datasource_parsers/tree/solr_phenodigm_1904) branch.
+### PhenoDigm
 
-The version in _master_ is an older version (October 2018) that **DOES NOT** call the IMPC SOLR API and it is **unlikely to work** but it has not been tested.
+Generates target-disease evidence querying the IMPC SOLR API.
 
-[solr_phenodigm](https://github.com/opentargets/evidence_datasource_parsers/tree/solr_phenodigm) is the version that Gautier handed over to the OT data team in February 2019. It *DOES* call the IMPC SOLR API but it has a number of bugs and **DOES NOT WORK**.
+#### System requirements and running time
+The PhenoDigm parser has been run both in a MacBook Pro and a Google Cloud machine. The current version is very memory greedy, using up to 60 GB. 
 
-**TODO**
-- [ ] map `intergenic` rsIDs to genes (~900k evidences)
-- [ ] improve mappings with manual curation
+* MacBook Pro: It takes around 8 hours to run in a 16 GB laptop.
+* Google Cloud Machine: It runs in around 2 hours in a 60 GB machine (_n1-standard-16_). There used to be such a machine set up but now one called _ag-ubuntu-20-04-lts_ in _open-targets-eu-dev_ can be used instead (see below). This machine has specs higher than needed (32 vCPU and 208 GB memory).
+
+#### Installation
+Before the parser can be run there is one dependency that needs to be installed manually:
+
+```sh
+# Create and activate virtual environment (requires python3)
+virtualenv -p python3 phenodigm_venv
+source phenodigm_venv/bin/activate
+
+# Install dependencies
+pip3 install -r requirements.txt
+
+# Manually download and install ontology-utils as it requires specific tag
+wget https://github.com/opentargets/ontology-utils/archive/bff0f189a4c6e8613e99a5d47e9ad4ceb6a375fc.zip
+pip3 install bff0f189a4c6e8613e99a5d47e9ad4ceb6a375fc.zip
+
+# Set environment variables
+export PYTHONPATH=.
+```
+Additionally, the parser needs access to Google Cloud, which requires downloading a key JSON file and setting the `GOOGLE_APPLICATION_CREDENTIALS` variable. Ask someone from the back-end team about it.
+
+#### Running MouseModels in a Google Cloud virtual machine
+
+Start the machine and connect to it via SSH.
+
+Before you start you may want to change the name of the output JSON by editing `MOUSEMODELS_EVIDENCE_FILENAME` variable in [settings.py](https://github.com/opentargets/evidence_datasource_parsers/blob/solr_phenodigm_1904/settings.py):
```

Comment:
```suggestion
Before you start you may want to change the name of the output JSON by editing `MOUSEMODELS_EVIDENCE_FILENAME` variable in [settings.py](https://github.com/opentargets/evidence_datasource_parsers/blob/solr_phenodigm_1904/settings.py):
```

I would suggest erasing this comment and change the Global variable in `settings.py` to include the current date automatically in this way, for example:

`MOUSEMODELS_EVIDENCE_FILENAME = f"phenodigm-{datetime.today().strftime('%Y-%m-%d')}.json"`

This would output, as of today: `phenodigm-2021-01-05.json`